update create NFT collection with LSP7 for v0.14.0 (#747)
lukso-network/docs: docs/learn/smart-contract-developers/create-nft-collection-lsp7.md

To create your custom contract based on the [LUKSO smart contracts](../../contracts/introduction.md), you will need the [`@lukso/lsp-smart-contracts`](../../tools/lsp-smart-contracts/getting-started.md) library. Go ahead and add it to your project:

```shell

> npm install @lukso/lsp-smart-contracts@0.13.0

Reviewer: same removing this please.

```suggestion
npm install @lukso/lsp-smart-contracts
```

```

## Create the Smart Contracts

When creating smart contracts representing digital assets on LUKSO, you will need to specify the token type and data keys for the 📄 [LSP4 Digital Asset Metadata](../../standards/tokens/LSP4-Digital-Asset-Metadata) that will be stored in the 🗂️ [ERC725Y](../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) storage of the Digital Asset. There are three different [token types](../../standards/tokens/LSP4-Digital-Asset-Metadata.md#lsp4tokentype):

- `0` = Token
- `1` = NFT
- `2` = Collection

> For this example we will use the `Token` token type. You can create a custom 🌄 [LSP7 Digital Asset Collection](../../standards/tokens/LSP7-Digital-Asset.md) that extends [LSP7Mintable](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) so that new assets can be created within the smart contract.

Reviewer: Suggest to remove the word collection here if it is a `Token` type used.

Also suggest to specify the feature of the preset `LSP7Mintable` that we are inheriting.

```suggestion
For this example we will use the `Token` token type. You can create a custom 🌄 [LSP7 Digital Asset Token](../../standards/tokens/LSP7-Digital-Asset.md) that extends [LSP7Mintable](../../contracts/contracts/LSP7DigitalAsset/presets/LSP7Mintable.md) so that new assets can be minted by the [`owner`](../../contracts/LSP7DigitalAsset/presets/LSP7Mintable#owner) of the smart contract.
```
